snippets/product-thumbnail.liquid

```diff
@@ -18,22 +18,23 @@
 
 <noscript>
   {%- if media.media_type == 'video' or media.media_type == 'external_video' -%}
-    <a href="{% if media.media_type == 'video' %}{{ media.sources[1].url }}{% else %}{{ media | external_video_url }}{% endif %}" aria-label="{{ 'products.product.video_exit_message' | t: title: product.title | escape }}">
-      <span class="product__media-icon motion-reduce">{% render 'icon-play' %}</span>
-      <div class="product__media media" style="padding-top: {{ 1 | divided_by: media.preview_image.aspect_ratio | times: 100 }}%;">
-        <img
-          srcset="{% if media.preview_image.width >= 288 %}{{ media.preview_image | img_url: '288x' }} 288w,{% endif %}
-                  {% if media.preview_image.width >= 576 %}{{ media.preview_image | img_url: '576x' }} 576w,{% endif %}
-                  {% if media.preview_image.width >= 550 %}{{ media.preview_image | img_url: '550x' }} 550w,{% endif %}
-                  {% if media.preview_image.width >= 1100 %}{{ media.preview_image | img_url: '1100x' }} 1100w{% endif %}"
-          src="{{ media | img_url: '550x550' }}"
-          sizes="(min-width: 1200px) calc((1200px - 10rem) / 2), (min-width: 750px) calc((100vw - 11.5rem) / 2), calc(100vw - 4rem)"
-          loading="lazy"
-          width="576"
-          height="{{ 576 | divided_by: media.preview_image.aspect_ratio | ceil }}"
-          alt="{{ media.preview_image.alt | escape }}"
-        >
-      </div>
+    <span class="product__media-icon motion-reduce">{% render 'icon-play' %}</span>
+    <div class="product__media media" style="padding-top: {{ 1 | divided_by: media.preview_image.aspect_ratio | times: 100 }}%;">
+      <img
+        srcset="{% if media.preview_image.width >= 288 %}{{ media.preview_image | img_url: '288x' }} 288w,{% endif %}
+                {% if media.preview_image.width >= 576 %}{{ media.preview_image | img_url: '576x' }} 576w,{% endif %}
+                {% if media.preview_image.width >= 550 %}{{ media.preview_image | img_url: '550x' }} 550w,{% endif %}
+                {% if media.preview_image.width >= 1100 %}{{ media.preview_image | img_url: '1100x' }} 1100w{% endif %}"
```

Comment:
Since this image is coming from the platform, I do think you're right that it's more of a platform issue.  Maybe we can speak to some folks on the team that handled the media integration to see if they can update it to request a better quality image.  It's surprising that Youtube defaults to 480 since that's quite an outdated size for any typical web application nowadays.

Reply:
@tylerrowsell do you have any context about this ? When using `media.preview_image` to grab the thumbnail from a youtube video it seems like it default to a low quality version. Unless I'm mistaking it seem to grab a `480px` wide one. 
When we could actually grab something a bit better and handle the size of the image being served with our `srcset` logic.
Curious to hear your thoughts on this when you're back :)

Reply:
Actually seems like it's my video that has a `hqdefault` image of 480px. https://img.youtube.com/vi/sELxgrgaQUk/hqdefault.jpg. 
But I think it still highlights that having a few more sizes in the scrset could be beneficial :)

Reply:
Interesting. This sounds like an issue that needs some changes on the platform side, like you mentioned.

I'll have a separate PR for media sizes this week so we can explore it there and create an issue.